__tests__/screens/BackupWords.test.tsx

```
@@ -0,0 +1,210 @@
+ */
+import React from 'react';
+import { render, fireEvent, act, waitFor } from '@testing-library/react-native';
+
+// The 24 test words — same as used in InitWallet.test.tsx
+const TEST_WORDS = 'abandon ability able about above absent absorb abstract absurd abuse access accident acid acoustic acquire across act action actor actress actual adapt add addict';
+const TEST_WORDS_ARR = TEST_WORDS.split(' ');
+
+// Mock lodash.shuffle to make tests deterministic.
+// We control which 5 word positions are selected for validation.
+const FIXED_INDEXES = [3, 8, 12, 18, 22]; // 1-indexed positions to validate
+jest.mock('lodash', () => {
```

ESLint

[error] 'jest' is not defined.

(no-undef)